service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellService.java

```diff
@@ -128,6 +127,35 @@ public static String getErrorMessage(List<FailureMessage> failureMessages) {
     }
   }
 
+  /**
+   * Cromwell accepts an object "Workflow Options" to specify additional configuration for a
+   * workflow. Here, we build that object with the parameters we care about. final_workflow_log_dir
+   * specifies the path where outputs will be written. write_to_cache and read_from_cache are both
+   * related to call caching. When a user enables call caching, these should both be set to true.
```

Comment:
Clarifying question: wouldn't setting `write_to_cache` to false confuse users that come from GCP Terra? I think in GCP Terra Rawls always sets `write_to_cache` to true and the call caching checkbox in UI refers to reading from call cache. So when unchecking the call caching box in Terra Azure they would assume that the `write_to_cache` will still be enabled even if read is disabled?

Reply:
This is a good catch. I think you're right that it makes sense to always `write_to_cache`. I'll make that swap.